blob/rollup.config.js

```diff
@@ -0,0 +1,61 @@
+import nodeResolve from 'rollup-plugin-node-resolve';
+import { uglify } from 'rollup-plugin-uglify';
+import replace from 'rollup-plugin-replace';
+import commonjs from 'rollup-plugin-commonjs';
+import shim from 'rollup-plugin-shim';
+
+export default [
+  {
+    external: ['ms-rest-js', 'crypto', 'fs', 'events', 'os'],
```

Comment:
Is it necessary to manually mark every node built-in module in use as external? :(

Reply:
I'm sure there are easier ways here, potentially with options to the node-resolve plugin, but this works for now.

Reply:
Cool. I'm picking up the task to start shipping rollup bundles in our core runtime and generated client libraries. Maybe I'll follow up if I learn something.

Reply:
Good luck! Definitely reach out if you have questions, and share your learnings 😄